### ISSUE TITLE
add filter for probability to vulcan

### ISSUE THREAD

**Issue by gilzimmerman** [RespirationScan/issues#25]
probability filter.

Can be same as hermes, or just minimum probability. Anything is fine.

**Comment by rs-tamar** [comment]
Yes, the Vulcan page will be a clone based on the Hermes just with the new and live data, so all of those kind of components

**Comment by rs-tamar** [comment]
This task is now completed
